tool/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	client "github.com/iotaledger/goshimmer/client"
+	"github.com/iotaledger/iota.go/trinary"
+)
+
+func testBroadcastData(api *client.GoShimmerAPI) (trinary.Hash, bool) {
+	txnHash, err := api.BroadcastData(txnAddr, txnData)
+	if err != nil {
+		fmt.Printf("Relay Check: broadcast data failed.\n %s\n", err)
+		return "", true
+	}
+	fmt.Printf("Relay Check: txnHash: %s\n", txnHash)
+	return txnHash, false
+}
+
+func testTargetGetTransactions(api *client.GoShimmerAPI, txnHash trinary.Hash) bool {
+	// query target node for broadcasted data
+	_, err := api.GetTransactions([]trinary.Hash{txnHash})
+	if err != nil {
+		fmt.Printf("Relay-Check: get transaction from local failed.\n")
+		return true
+	}
+	return false
+}
+
+func testNeighborGetTransactions(txnHash trinary.Hash) bool {
```

Comment:
Return an actual error from within this func, then print it out from the caller.